src/D2L.Security.OAuth2/Validation/Request/HttpRequestExtensions.cs

```diff
@@ -1,5 +1,4 @@
-#if !DNXCORE50
-
+﻿#if !NET5_0

```

Comment:
Entire file is excluded for .NET 5.0. Could do this as a condition in the csproj but that's maybe less obvious?

Reply:
Actually, I have an idea.